operate/manage-a-large-number-of-streaming-jobs.mdx

```diff
@@ -119,3 +119,4 @@ In some references, `/risingwave/bin/risingwave ctl scale horizon --include-work
 
 * Resources of meta store (etcd) and meta node: During recovering or scaling processes,there might be spikes in resource usage on the nodes. If you encounter OOM errors or observe some error logs in the meta such as `{"level":"ERROR","fields":{"message":"lease keeper failed","error":"grpc request error: status: Unavailable, message: \"etcdserver: request timed out, waiting for the applied index took too long\",}`, please try to scale up the nodes.
 * Resources of the prometheus or other monitoring systems: The number of metrics' time series grows linearly with the number of actors, so please pay attention to the resource requirements of your monitoring system and ensure that it can handle the growing number of metrics efficiently.
+* If you want to limit the parallelism for certain types of jobs, you can use session parameters like `streaming_parallelism_for_table` to set specific limits.
```

Comment:
Let's create a new section, under `## Tuning an existing cluster`, before `### Decrease the parallelism`.

We can title it: `### Limit Parallelism for less intensive Stream Jobs`

For the content, we can include:
1. A table, describing what each of these parameters do.
2. Examples, as per the PR description in: https://github.com/risingwavelabs/risingwave/pull/21366.
  We can just include `streaming_parallelism_for_table` and `streaming_parallelism_for_index`. Since these are the two most common ones.

Reply:
Bump